### ISSUE TITLE
Polish holding patterns

### ISSUE THREAD

**Issue by matjaeck** [OpenRA/OpenRA#16658]
After #16509 is merged, helicopters will wait just above a landed and rearming helicopter, instead of waiting somewhere next to the pad. From https://github.com/OpenRA/OpenRA/pull/16509#issuecomment-500165359:

> Could you polish this so that it waits a bit next to the pad?

> I think that can best be solved together with the other holding pattern polish issues in a separate PR.

<details>

![hoveroverusedpad](https://user-images.githubusercontent.com/37185497/59152954-36ec0e00-8a4f-11e9-976d-5bbec677afdb.gif)

**Comment by matjaeck** [comment]
Also applies to units that are waiting to resupply at a service depot.

**Comment by matjaeck** [comment]
Closing for the moment since the PR might get split and the issue is not present on bleed.
